CHANGELOG.md

```diff
@@ -85,6 +85,10 @@
 
 #### Bug Fixes
 
+* Fix identifier name false positive for enum cases with associated values in Xcode 10.
```

Comment:
requires two trailing spaces and lines should be limited to 80 chars as described in CONTRIBUTING.md: https://github.com/realm/SwiftLint/blob/master/CONTRIBUTING.md#tracking-changes